passkey-client/src/lib.rs

```diff
@@ -115,6 +115,17 @@ where
             tld_provider: Box::new(public_suffix::DEFAULT_PROVIDER),
         }
     }
+
+    /// Create a `Client` with a given `Authenticator` that uses the default
+    /// TLD verifier provided by `[public_suffix]` and with `allows_insecure_localhost`
+    /// set to `true`.
+    pub fn new_with_allows_localhost(authenticator: Authenticator<S, U>) -> Self {
+        Self {
+            authenticator,
+            allows_insecure_localhost: true,
+            tld_provider: Box::new(public_suffix::DEFAULT_PROVIDER),
+        }
+    }
```

Comment:
**Todo(Follow-up):** This made me realize I haven't updated this mirror in a while, I've added a builder method to set the `localhost` flag separately. This falls out of scope for this PR, I'll have a PR up tomorrow for this change.

Reply:
Checkout #7 and let me know if that works for you!